doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -563,6 +563,20 @@ for it.
 - `&x` as a Coq constr expression expands to
   `ltac2:(Control.refine (fun () => hyp @x))`.
 
+In the special case where Ltac2 antiquotations appear inside a Coq term
+notation, the notation variables are actually systematically bound in the body
+of the tactic expression with type `Ltac2.Init.preterm`. Such a type is meant
+to represent untyped syntactic Coq expressions, which can by typed in the
+current context using the `Ltac2.Constr.pretype` function.
+
+.. example::
+
+   The following notation is essentially the identity.
```

Comment:
Remove "essentially" or explain how it differs from the identity.

Reply:
It's not the identity, but I don't want to embark in a description of the internals of the pretyper...